src/plcc/load_spec/parse_spec/parse_lexical_spec.py

```diff
@@ -0,0 +1,67 @@
+from dataclasses import dataclass
+import re
+
+from ..load_rough_spec.parse_lines import Line
+
+@dataclass
+class LexicalRule:
+    line: Line
+    isSkip: bool
+    name: str
+    pattern: str
+
+@dataclass
+class LexicalSpec:
+    ruleList: [LexicalRule|Line]
+
+def parse_lexical_spec(lines):
+    lexical_spec = LexicalSpec([])
+    if lines == []:
+        return lexical_spec
+    if lines is None:
+        return lexical_spec
+
+    skipToken = re.compile(r'^skip\s+(?P<Name>\S+)\s+(?P<Pattern>((\'\S+\')|(\"\S+\")))\s*(?:#.*)*$')
+    tokenToken = re.compile(r'^token\s+(?P<Name>\S+)\s+(?P<Pattern>((\'\S+\')|(\"\S+\")))\s*(?:#.*)*$')
+    otherToken = re.compile(r'^(?P<Name>\S+)\s+(?P<Pattern>((\'\S+\')|(\"\S+\")))\s*(?:#.*)*$')
+    for line in lines:
+        l = line.string.strip()
+
+        #Skip blank or empty Lines
+        if l == '':
+            continue
+        if re.match(r'^\s*#.*$', l):
+            continue
+
+
+        skipMatches = re.match(skipToken, l)
+        if skipMatches:
+            newSkipRule = skipTokenGenerator(line, skipMatches['Name'], skipMatches['Pattern'])
+            lexical_spec.ruleList.append(newSkipRule)
+            continue
```

Comment:
@StoneyJackson 
I currently have this as one method for both, "skip_token_generates" and "regular_token_generates", since I have to match the regex again in the generator function anyways (to capture the 'name' and 'pattern'). Is this ok or would you like them separate like the example above?